andela/ah-backend-tesseract
# #161224772 redirect to frontend after account activation

#### What does this PR do?
modifies the backend to enable redirect after successful account activation.
#### Description of Task to be completed?
- Add redirect to frontend upon account activation
- Add likes and dislikes count to the article being returned
- Modify article update serializer to update the body using body not the description
#### How should this be manually tested?
- Create an account using a valid email.
- Activate your account by clicking on the link sent to your email
- You should be redirected to the frontend of the application after activation 
#### What are the relevant pivotal tracker stories?
[#161224772](https://www.pivotaltracker.com/story/show/161224772)
#### Screenshots (if appropriate)
<img width="1210" alt="screen shot 2018-10-16 at 09 02 15" src="https://user-images.githubusercontent.com/30045448/46995647-40241580-d122-11e8-9e24-f12fd2ea839d.png">
